src/oic/oauth2/__init__.py

```diff
@@ -410,7 +415,7 @@ def construct_AccessTokenRequest(
             request = self.message_factory.get_request_type("token_endpoint")
         if request_args is None:
             request_args = {}
-        if request is not ROPCAccessTokenRequest:
+        if request not in (ROPCAccessTokenRequest, CCAccessTokenRequest):
```

Comment:
If we follow the Type[x] annotation style above, this should allow for subclasses as well.